### DISCUSSION TITLE
How to resolve hostname in a WSL instance with internal IP

### DISCUSSION THREAD

**Discussion by ComputerDoktor** [alkampfergit/personal-blog#13] [General]
question asked by Gian Maria here
https://www.codewrecks.com/post/general/wsl-hostname/
Answer
Append domain mshome.net to your hostname, e.g. shismatrix.mshome.net

**Comment by proski** [comment]
For whatever reason, the name with "mshome.net" is resolving to some wrong IP address for me. However, I have found a working solution. If I append a dot to the full qualified name of the system (e.g. `mylaptop.mycompany.com.`), it resolves to the Windows host. Without the final dot, it resolves to `127.0.1.1`, which is the WSL system itself.
